WebRoot/skins/_base/base4/skin.css

```diff
@@ -236,6 +236,11 @@ BODY                                    {   margin:0px;     }
 #ZSplashPanel                           {   @SplashOuterBorder@     }
 #ZSplashBodyContainer                   {   @SplashInnerBorder@     }
 
+.console_inset_app_l {
```

Comment:
Not sure for this change. With this change, grey line shows up while dragging sash. (We had similar issue in past as well)

Reply:
Fixed by revisiting the solution
Now we are hiding sash container entirely and we will only show borders around folder tree or mail list, this way we will prevent the fluctuation when dragging sash